rpc/cmd.go

```diff
@@ -223,16 +235,21 @@ type ShortcutKeyCombo struct {
 func (CmdRsGetVoiceSettings) messageData() {}
 
 type CmdArgsSetVoiceSettings struct {
-	Input                *VoiceSettings     `json:"input"`
-	Output               *VoiceSettings     `json:"output"`
-	Mode                 *VoiceSettingsMode `json:"mode"`
-	AutomaticGainControl *bool              `json:"automatic_gain_control"`
-	EchoCancellation     *bool              `json:"echo_cancellation"`
-	NoiseSuppression     *bool              `json:"noise_suppression"`
-	QOS                  *bool              `json:"qos"`
-	SilenceWarning       *bool              `json:"silence_warning"`
-	Deaf                 *bool              `json:"deaf"`
-	Mute                 *bool              `json:"mute"`
+	Input                *VoiceSettingsIO   `json:"input,omitempty"`
+	Output               *VoiceSettingsIO   `json:"output,omitempty"`
+	Mode                 *VoiceSettingsMode `json:"mode,omitempty"`
+	AutomaticGainControl *bool              `json:"automatic_gain_control,omitempty"`
+	EchoCancellation     *bool              `json:"echo_cancellation,omitempty"`
+	NoiseSuppression     *bool              `json:"noise_suppression,omitempty"`
+	QOS                  *bool              `json:"qos,omitempty"`
+	SilenceWarning       *bool              `json:"silence_warning,omitempty"`
+	Deaf                 *bool              `json:"deaf,omitempty"`
+	Mute                 *bool              `json:"mute,omitempty"`
+}
+
+func (EmptyArgs) cmdArgs() {}
+
+type EmptyArgs struct {
```

Comment:
can't you use nil instead?

Reply:
Resolved in [ca68a83](https://github.com/dennisrijsdijk/disgo/commit/ca68a83591c1ade9626007093a8638ccebbde717)